### ISSUE TITLE
Model: Handling Predictions about Gait

### ISSUE THREAD

**Issue by ErPang97** [ErPang97/RunningGaitAnalysis#4]
So this is definitely an interesting question... how do we go about deciding what is considered a good gait vs. bad gait? Or is there something more quantifiable that we're going after?

Can maybe use ML?
- We would definitely need a training set, with features, and a label
- Follow up question, what are the features? Essentially, what would be the parameters, that the model uses, to make a decision on the label
- Since we're using data from a video, what about the keypoints, are we looking for exactly?
- Are we paying attention to, for example, the angles of the leg, during certain points of the running form? If we're looking at pronation, are we looking at the angle formed between one's feet, ankle and knee during the run motion? 
- Follow up question about the label, would this be a simple binary classification? Ie: good vs bad gait? some sort of regression? ie: a number that quantifies the gait somehow?
- Follow up, what library would we want to use? ie: Sklearn, PyTorch, Tensorflow?
